extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -9,6 +9,7 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
+import com.amazonaws.serverless.proxy.internal.LambdaContainerHandler;
```

Comment:
I'm not an expert on this one but `internal` sounds to be something that might get changed without any further deprecation phase.
Not completely sure but looking at the aws code it is actually nothing more than checking if the content type is either application/octet-stream, image/jpeg, image/png or image/gif

Reply:
That check comes from the java serverless project on amazon's github account.  It's probably safe enough.  I used it to mirror the same checks they do for their own integrations.

Reply:
In future versions please consider providing a new quarkus application property for binary content types apart from the default ones (application/octet-stream, image/jpeg, image/png or image/gif). Generating pdf is a real world use case (application/pdf )